docs/en/compute-edition/30/admin-guide/install/system-requirements.adoc

```diff
@@ -43,22 +40,24 @@ Each Defender requires 256MB of RAM and 8GB of host storage.
 
 The Defender uses `cgroups` v1 or v2 to cap resource usage at 512MB of RAM and 900 CPU shares where a typical load is ~1-5% CPU and 30-70MB RAM.
 
-The Defender stores its data in the `/var` folder.
-When allocating disk space for Defender, ensure the required space is available in the `/var` folder.
-Defenders are designed to be portable containers that collect data.
-Any data that must be persisted is sent to the Prisma Cloud Console for storage.
-Defenders don't require persistent storage.
-If you deploy persistent storage for Defenders, it can corrupt Defender files.
+For more than 10,000 Defenders you need 4 vCPUS and 10GB of RAM for every additional 5,000 Defenders For example, 20,000 connected Defenders require a total of 16 vCPUs, 50GB of RAM and 500GB SSD of persistent storage.
```

Comment:
For every additional 5,000 Defenders beyond 10,000 Defenders, you need 4 CPUs and 10GB RAM. For example, if there are 20,000 connected Defenders, you require 16 vCPUs, 50GB of RAM, and 500GB SSD of persistent storage.